.github/workflows/doxygen-gh-pages.yml

```diff
@@ -0,0 +1,17 @@
+name: Doxygen GitHub Pages Deploy Action
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: DenverCoder1/doxygen-github-pages-action@v1.3.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          config_file: doc/Doxyfile_bonus
```

Comment:
The Doxyfile doesn't have the _bonus suffix

Reply:
Nice catch - removed suffix